<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmovie</class>
 <widget class="QWidget" name="addmovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

color:white
</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Titre :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Réalisateur :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Durée :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Genre :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Date de sortie :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;
font: 700 28pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ajouter un film</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
